### PR TITLE
Update basic-bonfires.json

### PR DESCRIPTION
**Pull request by ChuanfengZhang**
correct a little mistake
Old say:Repeat a given string (first argument) num times (second argument). Return an empty string if num is not a positive number.
The first "num" should be "str".

### CODE REVIEW

**Review by MarcLBryson — APPROVED** [review]

**Comment by Bouncey** [comment]
I don't remember seeing this as an issue.

Your edit confuses the instructions.

Could we use better/different grammar for non-native English speakers? 

Will this potentially be resolved in beta with language selection?

/cc @FreeCodeCamp/issue-moderators

**Comment by ChuanfengZhang** [comment]
Sorry,I mistaken it.
@Bouncey

**Comment by abhisekp** [comment]
@ChuanfengZhang @Bouncey How about the following?

> Repeat a given string *`str`* (first argument) for *`num`* times (second argument).

**Comment by Bouncey** [comment]
@abhisekp 
### LGTM :+1:

**Comment by Bouncey** [comment]
@ChuanfengZhang please ammend your PR

Don't forget to use `git commit --amend` so as not to add an additional commit

File: seed/challenges/01-front-end-development-certification/basic-bonfires.json [hunk 1]
```diff
@@ -400,7 +400,7 @@
       "id": "afcc8d540bea9ea2669306b6",
       "title": "Repeat a string repeat a string",
       "description": [
-        "Repeat a given string (first argument) <code>num</code> times (second argument). Return an empty string if <code>num</code> is not a positive number.",
+        "Repeat a given string (first argument) <code>str</code> times (second argument). Return an empty string if <code>num</code> is not a positive number.",
```
Comment (abhisekp) [inline seed/challenges/01-front-end-development-certification/basic-bonfires.json]: > Repeat a given string *`str`* (first argument) for *`num`* times (second argument).

**Review by abhisekp — CHANGES_REQUESTED** [review]
Modify instruction.

**Comment by Bouncey** [comment]
@ChuanfengZhang Do you have any progress on this?

**Comment by hallaathrad** [comment]
Do we have a policy on completing the changes? I'm up for it.

**Comment by ChuanfengZhang** [comment]
@Bouncey Sorry,I a starter.Is it OK now?

**Comment by Bouncey** [comment]
## LGTM :+1: 

Don't worry about it being your first time, you did great!

Thanks for contributing :tada:

**Pull request merged**